### PR TITLE
Various small improvements

### PR DESCRIPTION
**Pull request by jwillemsen**
Creating now a PR to get the new test and some other changes tested on the various CI systems. This PR includes the changes of #492. At the moment #492 has been merged I will merge master again in this branch so that we just get the new test and some other minor changes.

### CODE REVIEW

**Comment by jwillemsen** [comment]
The failure of tests/DCPS/SampleLost/run_test.pl is intended, this is a new test that reproduces a crash we see. When this PR and #490 are merged this test runs

**Comment by mitza-oci** [comment]
After PR #528 is merged (that's a branch I created by just cherry-picking the new test from this one), we'll revisit this PR to get all the other cleanup merged as well.

**Comment by jwillemsen** [comment]
Thanks

File: examples/DCPS/Messenger_IOGR_Imr/README [hunk 1]
```diff
@@ -5,5 +5,5 @@ Status: Still under development.
 
 This test incarnates multiple instances of ImRified InfoRepo's and
 ties them all together into a single IOGR. This can be used as
-a FT startgy for clients to autamtically swicth over to a backup
+a FT start for clients to automatically switch over to a backup
```
Comment (mitza-oci) [inline examples/DCPS/Messenger_IOGR_Imr/README]: Wouldn't this word be "strategy" and not "start"?

Comment (jwillemsen) [inline examples/DCPS/Messenger_IOGR_Imr/README]: Yes, fixed

**Pull request merged**